doc-content/enterprise-only/openshift/operator-modify-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='operator-modify-proc']
+= Modifying an environment that is deployed using operators
+
+If an environment is deployed using operators, you can not modify it using typical OpenShift methods. For example, if you delete a pod, it is re-created automatically with the same parameters.
+
+To modify the environment, you must modify the YAML description of the environment. You can change common settings, such as passwords, add new {KIE_SERVERS}, and scale {KIE_SERVERS}.
+
+.Procedure
+. Enter your project in the OpenShift Web cluster console. 
+. In the OpenShift Web console navigation panel, select *Operators* and then *Subscriptions*.
+. Click the name of the subscription that containt `businessautomation`. Information about this subscription is displayed.
+. Under the *Installed version* heading, click the version name of the subscription. An overview of the subscription is displayed.
+. Select the *Instances* tab.
+. Click the name of a deployed environment. 
+. Select the *YAML* tab. A YAML source is displayed.
+. If you want to change common settings, such as passwords, edit the values under `commonConfig:`.
+. If you want to add new {KIE_SERVERS}, add their descriptions at the end of the block under `servers:`. For example:
++ 
+** To add two servers named `server-a` and `server-a-2`, add the following lines:
++
+[subs="attributes,verbatim,macros"]
+----
+- deployments: 2
+  name: server-a
+----
++ 
+** To add an immutable {KIE_SERVER} that includes services built from source in an S2I process, add the following lines:
++
+[subs="attributes,verbatim,macros"]
+----
+- build:
+    kieServerContainerDeployment: <deployment>
+    gitSource:
+      uri: <url>
+      reference: <branch>
+      contextDir: <directory>
+----
++
+Replace the following values:
++
+*** `<deployment>`: The identifying information of the decision service (KJAR file) that is built from your source. The format is: `<containerId>=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`. The Maven build process must produce all these files from the source in the Git repository.
```

Comment:
Same comment as mine before, about "format is `<container...`" and "for example, `container...`"